docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,145 @@
+
+.NET Core implements the .NET Standard Library, and therefore supports .NET Standard Libraries.
+
+Workloads
+---------
+
+.NET Core includes a single application model: console apps. That's useful for tools, local services and console games. Additional application models build on top of .NET Core, such as: 
+
+- [ASP.NET Core](http://asp.net)
+- [Windows 10 UWP](https://developer.microsoft.com/windows)
+- [Xamarin.Forms](https://www.xamarin.com/forms)
+
+Open Source
+-----------
+
+[.NET Core](https://github.com/dotnet/core) is open source (MIT license) and was contributed to the [.NET Foundation](http://dotnetfoundation.org) by Microsoft in 2014. It is now one of the most active .NET Foundation projects. It can be freely adopted by individuals and companies, including for personal, academic or commercial purposes. Multiple companies use .NET Core as part of apps, tools, new platforms and hosting services. Some of these companies make significant contributions to .NET Core on GitHub and provide guidance on the product direction as part of the [.NET Foundation Technical Steering Group](http://www.dotnetfoundation.org/blog/tsg-welcome).
+
+Acquisition
+===========
+
+.NET Core is distributed in two main ways, as packages on NuGet.org and as standalone distributions.
+
+Packages
+--------
+
+- [.NET Core Packages](packages.md) contain the .NET Core runtime and libraries (reference assemblies and implementations). For example, [System.Net.Http](https://www.nuget.org/packages/System.Net.Http/).
+- [.NET Core Metapackages](packages.md) describe various layers and app-models by referencing the appropriate set of versioned library packages.
+
+Distributions
+-------------
+
+You can download .NET Core at the [.NET Core Getting Started](https://www.microsoft.com/net/core) page.
+
+- The *Microsoft .NET Core* distribution includes the CoreCLR runtime, associated libraries, a console application host and the `dotnet` app launcher. It is described by the [`Microsoft.NETCore.App`](https://www.nuget.org/packages/Microsoft.NETCore.App) metapackage.
+- The *Microsoft .NET Core SDK* distribution includes .NET Core and a set of tools for restoring NuGet packages and compiling and building apps. 
+
+Typically, you will first install the .NET Core SDK to get started with .NET Core development. You may choose to install additional .NET Core (perhaps pre-release) builds. The `dotnet new` command will 
+
+Architecture
+============
+
+.NET Core is a cross-platform .NET implementation. The primary architectural concerns unique to .NET Core are related to providing platform-specific implementations for supported platforms.
+
+Environments
+------------
+
+.NET Core is supported by Microsoft on Windows, OS X and Linux. On Linux, Microsoft primarily supports .NET Core running on Red Hat Enterprise Linux (RHEL) and Debian distribution families.
```

Comment:
This is true, yet also very awkwardly put. I would simply link to the OS matrix. Also, there should be an explanation of supported vs. run on. As in, Ubuntu LTS releases are supported and you can dial CSS if you have the contract, while Linux Mint will work, as in, you will be able to run the SDK and your apps on it, but don't call CSS if stuff explodes.